sdk/storage/azure-storage-file/README.md

```diff
@@ -1,207 +1,215 @@
+2. To use a storage account [shared access key](https://docs.microsoft.com/rest/api/storageservices/authenticate-with-shared-key/),
+   provide the key as a string.
 
-Once you have your account URL and credentials ready, you can create the FileServiceClient:
+#### Creating the client from a connection string
+Depending on your use case and authorization method, you may prefer to initialize a client instance with a storage
+connection string instead of providing the account URL and credential separately. To do this, pass the storage
+connection string to the client's `from_connection_string` class method:
 
 ```python
 from azure.storage.file import FileServiceClient
 
-service = FileServiceClient(account_url="https://<my-storage-account-name>.file.core.windows.net/", credential=credential)
+service = FileServiceClient.from_connection_string(conn_str="my_connection_string")
 ```
 
 ## Key concepts
-
-File storage includes the following concepts:
-* The storage account
-* A file storage share
-* An optional hierarchy of directories
-* A file in the share which may be up to 1 TiB in size
-
-#### Clients
-
-The Storage File SDK provides four different clients to interact with the File Service:
-1. **FileServiceClient** - this client interacts with the File Service at the account level. 
-    It provides operations to retrieve and configure the service properties
-    as well as list, create, and delete shares within the storage account.
-    For operations relating to a specific share, a client for that entity
-    can also be retrieved using the `get_share_client` function.
-2. **ShareClient** - this client represents interaction with a specific
-    file share, although that share need not exist yet. It provides operations to create, delete, or
-    configure shares and includes operations to list and create files or directories.
-    For operations relating to a specific directory or file, those clients can also be retrieved using
-    the `get_directory_client` or `get_file_client` functions.
-3. **DirectoryClient** - this client represents interaction with a specific
-    directory, although that directory need not exist yet. It provides operations to create, delete, and list 
-    directories and subdirectories, as well as create and delete files in the directory. For operations 
-    relating to a specific subdirectory or file, a client for that entity can also be retrieved using
-    the `get_subdirectory_client` and `get_file_client` functions.
-4. **FileClient** - this client represents interaction with a specific file, although the file need not
-    exist yet. It provides operations to create, upload, copy, and download files as well as more advanced
-    operations.
+The following components make up the Azure File Service:
+* The storage account itself
+* A file share within the storage account
+* An optional hierarchy of directories within the file share
+* A file within the file share, which may be up to 1 TiB in size
```

Comment:
nit: In the blob docs it denotes things as `"TB"` not `"TiB"

Reply:
You're awesome with these small discrepancies! However https://docs.microsoft.com/en-us/azure/storage/files/storage-files-planning seems to indicate 1 TiB is correct.

Reply:
Also FWIW, I'm not sure if the blobs docs is correct or not. https://docs.microsoft.com/en-us/azure/storage/common/storage-scalability-targets#azure-blob-storage-scale-targets says it's measured in TiB, whereas https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-block-blobs says it's TB. I have no idea which one is correct, but I would guess that TiB is probably how we actually measure things.

Reply:
Going to assume the limits described in the storage-scalability-targets doc are correct.

Reply:
It's probably impossible to state whether anything that uses "TB" is technically correct or not since some people use "TB" to mean 1000^4 bytes and some people use it to mean 1024^4 bytes (and even some weird systems that do things like 1 TB = 1000 GiB 😱 ) and regardless of your intent you have no control over your audiences interpretation. I was mostly asking after consistency, and in particular maybe to use TiB elsewhere, to match the more specific units here (assuming that that is actually correct)